gh_actions_exporter/metrics.py

```diff
@@ -74,14 +95,39 @@ def set_status(self, webhook: WebHook) -> str:
             status=webhook.workflow_run.status,
             repo=webhook.repository.full_name,
             created_at=webhook.workflow_run.created_at,
+            finished_at=finished_at
         ).state(status)
         return status
 
     def cleaner(self):
```

Comment:
This function doesn't look like cleaning all data

Reply:
The goal of this function is not to clean all data, but rather to clean completed builds. Should we rather rename it to something that represent this function?

Reply:
Yep, having clear name function is really  important.